src/SettingsSectionAccount/index.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import classNames from 'classnames';
+import SectionHeaderGeneral from '../SectionHeaderGeneral';
+import SettingsItemEmail from '../SettingsItemEmail';
+import SettingsItemPassword from '../SettingsItemPassword';
+import moment from 'moment';
+import Wrapper from '../Wrapper';
+import Typography from '../Typography';
+import Spacer from '../Spacer';
+import styles from './styles.css';
+const PropTypes = React.PropTypes;
+
+function SettingsSectionAccount(props) {
+  return (
+    <div className={styles.outerWrapper}>
+      <SectionHeaderGeneral
+        title={props.title}
+        />
+      <SettingsItemEmail
+        email={props.email}
+        emailOnClick={props.emailOnClick}
+        />
+      <SettingsItemPassword
+        className={classNames({[props.className]: props.className})}
+        confirmNewPassword={props.confirmNewPassword}
+        expanded={props.expanded}
+        inactive={props.inactive}
+        newPassword={props.newPassword}
+        oldPassword={props.oldPassword}
+        onBlur={props.onBlur}
+        onChange={props.onChange}
+        onClick={props.onClick}
+        onCloseClick={props.onCloseClick}
+        onFocus={props.onFocus}
+        onRadioButtonClicked={props.onRadioButtonClicked}
```

Comment:
onRadioButtonClicked is an error inside SettingsItemPassword. You can remove from here and the PropType validation on this component and SettingsItemPassword.